chainercv/datasets/voc/voc_detection_dataset.py

```diff
@@ -124,7 +125,12 @@ def get_example(self, i):
             label.append(voc_utils.voc_detection_label_names.index(name))
         bbox = np.stack(bbox).astype(np.float32)
         label = np.stack(label).astype(np.int32)
-        difficult = np.array(difficult, dtype=np.bool)
+        if self.use_difficult:
```

Comment:
Are you saying that `np.array(difficult, dtype=bool)` should be called after `if return_difficult`?